### DISCUSSION TITLE
academy/web-scraping-for-beginners/data-extraction/node-continued

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1057] [Comments]
# academy/web-scraping-for-beginners/data-extraction/node-continued

Continue learning how to create a web scraper with Node.js and Cheerio. Learn how to parse HTML and print the results of the data your scraper has collected.

https://docs.apify.com/academy/web-scraping-for-beginners/data-extraction/node-continued

**Comment by AmmarSalmi** [comment]
```
// This code will only work in the browser, and NOT in Node.js
const results = [];

for (const product of products) {
    const titleElement = product.querySelector('a.product-item__title');
    const title = titleElement.textContent.trim();

    const priceElement = subwoofer.querySelector('span.price');
    const price = priceElement.childNodes[2].nodeValue.trim();

    results.push({ title, price });
}
```
Subwoofer here is a typo. Should be product
